aemedge/blocks/accordion/accordion.css

```diff
@@ -76,8 +76,16 @@
   padding: 0.5rem 0;
 }
 
+.accordion .block {
+  margin-left: -16px !important;
+}
```

Comment:
Doing this is affecting all instances of accordions, including the standalone one (not in tabs). So instead of doing a negative margin on the .accordion .block, instead you should decrease the spacing that the TABS adds around it.
![image](https://github.com/user-attachments/assets/154ca5d4-77fb-4b32-91dc-f3b550b0ba04) (from https://www.sling.com/whatson/sports/nfl/stream-nfl-playoffs-live)